Client/Application/LeanplumIntegration.swift

```diff
@@ -173,6 +175,13 @@ class LeanplumIntegration {
         return !UIApplication.shared.canOpenURL(klar)
     }
 
+    func canInstallPocket() -> Bool {
+        guard let pocket = URL(string: "pocket://") else {
+            return false
+        }
+        return !UIApplication.shared.canOpenURL(pocket)
```

Comment:
For this to work, the `pocket://` url scheme also needs to be added to the `Info.plist` like you did for the Focus and Klar checks.